tests/documents.test.ts

```diff
@@ -780,6 +783,41 @@ Hint: It might not be working because maybe you're not up to date with the Meili
         expect(index.primaryKey).toEqual(null);
         expect(task.status).toEqual('failed');
       });
+
+      test(`${permission} key: test updateDocumentsByFunction`, async () => {
+        const client = await getClient(permission),
+          index = client.index<(typeof dataset)[number]>(indexPk.uid),
+          adminKey = await getKey('Admin');
+
+        client.waitForTask(
+          (await index.updateFilterableAttributes(['id'])).taskUid,
+        );
+
+        await fetch(`${HOST}/experimental-features`, {
+          body: JSON.stringify({ editDocumentsByFunction: true }),
+          headers: {
+            Authorization: `Bearer ${adminKey}`,
+            'Content-Type': 'application/json',
+          },
+          method: 'PATCH',
+        });
+
+        await index.waitForTask((await index.addDocuments(dataset)).taskUid);
+
+        await client.waitForTask(
+          (
+            await index.updateDocumentsByFunction({
+              context: { ctx: 'Harry' },
+              filter: 'id = 4',
+              function: 'doc.comment = `Yer a wizard, ${context.ctx}!`',
+            })
+          ).taskUid,
+        );
+
+        const doc = await index.getDocument(4);
+
+        expect(doc).toHaveProperty('comment', 'Yer a wizard, Harry!');
```

Comment:
Just rewriting it a bit for readability and consistency 👍 

Would be amazing if you could do the same for the other tests you added 🙏 
```suggestion
  const client = await getClient(permission);
        const index = client.index<(typeof dataset)[number]>(indexPk.uid);
        const adminKey = await getKey('Admin');

        const { taskUid: updateFilterableAttributesTaskUid } =
          await index.updateFilterableAttributes(['id']);
        await client.waitForTask(updateFilterableAttributesTaskUid);

        await fetch(`${HOST}/experimental-features`, {
          body: JSON.stringify({ editDocumentsByFunction: true }),
          headers: {
            Authorization: `Bearer ${adminKey}`,
            'Content-Type': 'application/json',
          },
          method: 'PATCH',
        });

        const { taskUid: addDocumentsTaskUid } =
          await index.addDocuments(dataset);
        await index.waitForTask(addDocumentsTaskUid);

        const { taskUid: updateDocumentsByFunctionTaskUid } =
          await index.updateDocumentsByFunction({
            context: { ctx: 'Harry' },
            filter: 'id = 4',
            function: 'doc.comment = `Yer a wizard, ${context.ctx}!`',
          });

        await client.waitForTask(updateDocumentsByFunctionTaskUid);

        const doc = await index.getDocument(4);

        expect(doc).toHaveProperty('comment', 'Yer a wizard, Harry!');
```